storage/bytes.go

```diff
@@ -0,0 +1,42 @@
+package storage
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	BYTE     = 1.0
+	KILOBYTE = 1024 * BYTE
+	MEGABYTE = 1024 * KILOBYTE
+	GIGABYTE = 1024 * MEGABYTE
+	TERABYTE = 1024 * GIGABYTE
+)
+
+func byteSize(bytes int64) string {
```

Comment:
There is a common package for this in the Go world. I'd probably remove this file in favor of https://github.com/dustin/go-humanize